### PR TITLE
URL changed

### PR DESCRIPTION
**Pull request by tanay13**

### CODE REVIEW

**Comment by tanay13** [comment]
@YashKumarVerma just check if this is right

File: README.md [hunk 1]
```diff
@@ -1,14 +1,14 @@
 ## TypeScript Starter Template
 
-[![Build Status](https://travis-ci.com/YashKumarVerma/typescript-template.svg?branch=master)](https://travis-ci.com/YashKumarVerma/typescript-template)
-[![codecov](https://codecov.io/gh/YashKumarVerma/typescript-template/branch/master/graph/badge.svg)](https://codecov.io/gh/YashKumarVerma/typescript-template)
+[![Build Status](https://travis-ci.com/YashKumarVerma/typescript-template.svg?branch=master)](https://img.shields.io/github/languages/top/yashkumarverma/typescript-template)
+[![codecov](https://codecov.io/gh/YashKumarVerma/typescript-template/branch/master/graph/badge.svg)](https://img.shields.io/github/languages/top/yashkumarverma/typescript-template)
```
Comment (YashKumarVerma) [inline README.md:4]: this has to be changed to lockedup-coders/nostalgify

**Comment by YashKumarVerma** [comment]
[Build Status](https://travis-ci.com/YashKumarVerma/typescript-template)

should become

[Build Status](https://travis-ci.com/LockedUp-Coders/nostalgify)
and so on

**Comment by tanay13** [comment]
should i remove

**Pull request closed without merging**